packages/esm-service-queues-app/src/queue-services/queue-service-form.workspace.tsx

```diff
@@ -170,8 +164,8 @@ const QueueServiceForm: React.FC<QueueServiceFormProps> = ({ closePanel }) => {
           </Layer>
         </Column>
       </Stack>
-      <ButtonSet className={isTablet ? styles.tablet : styles.desktop}>
-        <Button className={styles.button} kind="secondary" onClick={() => closePanel()}>
+      <ButtonSet className={styles.buttonSet}>
+        <Button className={styles.button} kind="secondary" onClick={() => closeWorkspace()}>
```

Comment:
```suggestion
        <Button className={styles.button} kind="secondary" onClick={closeWorkspace}>
```

nit: Since it doesn't take any args.